plugins/relicscape

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/IdylRS/RelicScape.git
+commit=7c145c0572b177e6f84c7a3f064358d8114a3f22
+authors=Idyl
```

Comment:
I'll just remove this for now, as it's meant to reference GitHub usernames, not ingame/other aliases. You yourself will always be considered an author based on your GitHub username from the `repository` field.
```suggestion
```